### DISCUSSION TITLE
How to change content in the mgt-person-card

### DISCUSSION THREAD

**Discussion by widarodd** [microsoft-search/pnp-modern-search#3688] [Q&A]
I can't find any examples on how to remove or add elements in the mgt-person-card.
Is it possible to configure it?
I want to remove the phone icon and the QuickMessage tab.
![image](https://github.com/microsoft-search/pnp-modern-search/assets/163286044/8707a2e2-4ca5-45d7-8c2b-11f888f1e32c)
I'm using HTML and HandleBars.

**Comment by wobba** [comment]
See https://learn.microsoft.com/en-us/graph/toolkit/components/person-card
